src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataService.scala

```diff
@@ -138,6 +138,8 @@ object FileMetadataService {
   val ClosureStartDate = "ClosureStartDate"
   val Filename = "Filename"
   val FileType = "FileType"
+  val FileReference = "FileReference"
```

Comment:
Realise this is nothing to do with your PR. But could you add a unit test to check all these static values in `FileMetadataService` object? Good practice in case someone inadvertently changes them, at least they'll be a unit test warning.